{
  "comments": [
    {
      "key": {
        "uuid": "a9f05bea_cb710e06",
        "filename": "rw_core/core/grpc_nbi_api_handler.go",
        "patchSetId": 3
      },
      "lineNbr": 867,
      "author": {
        "id": 1000003
      },
      "writtenOn": "2019-08-19T23:20:22Z",
      "side": 1,
      "message": "is this really an error, i.e. it seems like an expected case, in which case it is not an error",
      "revId": "881e7ea14e8c2e9bd9a6b266152367d50bba23ed",
      "serverId": "2a2bfe1b-c5c2-48ed-9ac8-16438ab24388",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "3816ef3b_7281a4f1",
        "filename": "rw_core/core/grpc_nbi_api_handler.go",
        "patchSetId": 3
      },
      "lineNbr": 867,
      "author": {
        "id": 1000015
      },
      "writtenOn": "2019-08-19T23:22:07Z",
      "side": 1,
      "message": "Error. err !\u003d nil :) This was the failure you were hitting in the logs for the last invalid stream",
      "parentUuid": "a9f05bea_cb710e06",
      "revId": "881e7ea14e8c2e9bd9a6b266152367d50bba23ed",
      "serverId": "2a2bfe1b-c5c2-48ed-9ac8-16438ab24388",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "8cc757a4_7bb4746b",
        "filename": "rw_core/core/grpc_nbi_api_handler.go",
        "patchSetId": 3
      },
      "lineNbr": 867,
      "author": {
        "id": 1000003
      },
      "writtenOn": "2019-08-19T23:25:02Z",
      "side": 1,
      "message": "my point being an error in the log typically means something has gone horribly wrong and the system is no longer guaranteed to work. Given that we are handling the error, it is not propagated, etc. I am not sure this is \"error log worthy\" instead this is an expected and handled situation.\n\nwhen an error is in the log the operator panics, as it is handled and expected should we really cause a panic?",
      "parentUuid": "3816ef3b_7281a4f1",
      "revId": "881e7ea14e8c2e9bd9a6b266152367d50bba23ed",
      "serverId": "2a2bfe1b-c5c2-48ed-9ac8-16438ab24388",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "b1e11b1f_3eb20ae8",
        "filename": "rw_core/core/grpc_nbi_api_handler.go",
        "patchSetId": 3
      },
      "lineNbr": 867,
      "author": {
        "id": 1000015
      },
      "writtenOn": "2019-08-19T23:34:11Z",
      "side": 1,
      "message": "Its not an expected error. A grpc failure is something to be scared about. It won\u0027t come now because of the fix but if it does come as a valid grpc failure, its better to be aware about the failure of grpc. I am also thinking if the severity is right. As of now, I think its ok. If you want to lower, we can move it to Info also.",
      "parentUuid": "8cc757a4_7bb4746b",
      "revId": "881e7ea14e8c2e9bd9a6b266152367d50bba23ed",
      "serverId": "2a2bfe1b-c5c2-48ed-9ac8-16438ab24388",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "e2b92c54_38cc5ffd",
        "filename": "rw_core/core/grpc_nbi_api_handler.go",
        "patchSetId": 3
      },
      "lineNbr": 911,
      "author": {
        "id": 1000003
      },
      "writtenOn": "2019-08-19T23:20:22Z",
      "side": 1,
      "message": "same as above",
      "revId": "881e7ea14e8c2e9bd9a6b266152367d50bba23ed",
      "serverId": "2a2bfe1b-c5c2-48ed-9ac8-16438ab24388",
      "unresolved": false
    }
  ]
}